subsys/bluetooth/Kconfig.logging

```diff
@@ -28,267 +28,344 @@ parent-module = BT
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_ASCS
 
+if BT_BAP_UNICAST_SERVER
 module = BT_BAP_UNICAST_SERVER
 module-str = "Bluetooth Audio Unicast Server"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_BAP_UNICAST_SERVER
 
+if BT_BAP_UNICAST_CLIENT
 module = BT_BAP_UNICAST_CLIENT
 module-str = "Basic Audio Profile"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_BAP_UNICAST_CLIENT
 
+if BT_BAP_BROADCAST_SOURCE
 module = BT_BAP_BROADCAST_SOURCE
 module-str = "Bluetooth Audio Broadcast Source"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_BAP_BROADCAST_SOURCE
 
+if BT_BAP_BROADCAST_SINK
 module = BT_BAP_BROADCAST_SINK
 module-str = "Bluetooth Audio Broadcast Sink"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_BAP_BROADCAST_SINK
 
+if BT_BAP_SCAN_DELEGATOR
 module = BT_BAP_SCAN_DELEGATOR
 module-str = "Broadcast Audio Scan Service"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_BAP_SCAN_DELEGATOR
 
+if BT_BAP_BROADCAST_ASSISTANT
 module = BT_BAP_BROADCAST_ASSISTANT
 module-str = "Broadcast Audio Scan Service client debug"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_BAP_BROADCAST_ASSISTANT
 
+if BT_BAP_STREAM
 module = BT_BAP_ISO
 module-str = "Bluetooth Audio ISO"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_BAP_STREAM
 
 # CAP
 
+if BT_CAP_ACCEPTOR
 module = BT_CAP_ACCEPTOR
 module-str = "Common Audio Profile Acceptor"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_CAP_ACCEPTOR
 
+if BT_CAP_INITIATOR
 module = BT_CAP_INITIATOR
 module-str = "Common Audio Profile Initiator"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_CAP_INITIATOR
 
+if BT_CAP_COMMANDER
 module = BT_CAP_COMMANDER
 module-str = "Common Audio Profile Commander"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_CAP_COMMANDER
 
+if BT_AUDIO
 module = BT_CAP_COMMON
 module-str = "Common Audio Profile Common"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_AUDIO
 
+if BT_CAP
 module = BT_CAP_STREAM
 module-str = "Common Audio Profile Stream"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_CAP
 
 # CSIP
 
+if BT_CSIP_SET_MEMBER
 module = BT_CSIP_SET_MEMBER
 module-str = "Coordinated Set Identification Service"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_CSIP_SET_MEMBER
 
+if BT_CSIP_SET_COORDINATOR
 module = BT_CSIP_SET_COORDINATOR
 module-str = "Coordinated Set Identification Profile Set Coordinator"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_CSIP_SET_COORDINATOR
 
+if BT_AUDIO
 module = BT_CSIP_SET_MEMBER_CRYPTO
 module-str = "Coordinated Set Identification Profile crypto functions"
 source "subsys/logging/Kconfig.template.log_config_inherit"
+endif # BT_AUDIO
```

Comment:
Can we do an || / "OR" here? 

Should only be available if `BT_CSIP_SET_MEMBER` or `BT_CSIP_SET_COORDINATOR`

Reply:
Could you make a hidden kconfig for this then?